app/crud/user.py

```diff
@@ -0,0 +1,35 @@
+from sqlalchemy.orm import Session
+from app.models.user import User
+from app.schemas.user import UserCreate, UserUpdate
+
+
+class CRUDUser:
+    def get_user(self, db: Session, user_id: int):
+        return db.query(User).filter(User.id == user_id).first()
+
+    def get_users(self, db: Session, skip: int = 0, limit: int = 100):
+        return db.query(User).offset(skip).limit(limit).all()
+
+    def create_user(self, db: Session, user: UserCreate):
+        db_user = User(**user.dict())
+        db.add(db_user)
+        db.commit()
+        db.refresh(db_user)
+        return db_user
+
+    def update_user(self, db: Session, user_id: int, user: UserUpdate):
+        db_user = db.query(User).filter(User.id == user_id).first()
+        for field, value in user.dict().items():
```

Comment:
Instead of looping through items you could use something like that, but test it first

    update_data = user_update.dict(exclude_unset=True)
    db.query(User).filter(User.id == user_id).update(update_data)